libs/community/pyproject.toml

```diff
@@ -92,6 +92,7 @@ hdbcli = {version = "^2.19.21", optional = true}
 oci = {version = "^2.119.1", optional = true}
 rdflib = {version = "7.0.0", optional = true}
 nvidia-riva-client = {version = "^2.14.0", optional = true}
+unstructured = {version = "^0.12.4", optional = true, python = ">=3.9.0,<3.12"}
```

Comment:
Could you add this to the extended_testing group below in the tool.poetry.extras below?

Reply:
@eyurtsev it's already there, or am I missing anything else?

https://github.com/langchain-ai/langchain/blob/143d0077d8e8ddb8ea4cab8b665f673d8bab2d48/libs/community/pyproject.toml#L261

Reply:
Ah, looks like I forgot to regenerate the lockfile after adding the extra, just pushed again :+1: